samples/performance/multithreading_render_passes/multithreading_render_passes.cpp

```diff
@@ -52,7 +52,7 @@ bool MultithreadingRenderPasses::prepare(vkb::Platform &platform)
 		shadow_render_targets[i] = create_shadow_render_target(SHADOWMAP_RESOLUTION);
 	}
 
-	load_scene("scenes/sponza/Sponza01.gltf");
+	load_scene("scenes/bonza/Bonza4x.gltf");

```

Comment:
This is still wrong I'm afraid. It should be Bonza4X not Bonza4x (upper vs lower case x)

Reply:
Fixed :)

Reply:
Fixed :)